sections_flask-jwt-extended/models/item.py

```diff
@@ -16,12 +17,21 @@ def __init__(self, name, price, store_id):
         self.store_id = store_id
 
     def json(self):
-        return {'name': self.name, 'price': self.price}
+        return {
+            'id': self.id,
+            'name': self.name,
+            'price': self.price,
+            'store_id': self.store_id
+        }
 
     @classmethod
     def find_by_name(cls, name):
         return cls.query.filter_by(name=name).first()
 
+    @classmethod
+    def find_all(cls):
+        return cls.query.all()
+
```

Comment:
I prefer to keep resources unaware of what's happening internally, so let's put this code in the model instead.